Source/WebInspectorUI/UserInterface/Views/Popover.js

```diff
@@ -203,7 +203,7 @@ WI.Popover = class Popover extends WI.Object
 
     // Private
 
-    static _getCanvasContext(width, height)
+    static _getCanvasContext(width, height, scaleFactor)
```

Comment:
NIT: instead of requiring the client to provide this perhaps we should just hardcode `window.devicePixelRatio` in this function since it should really be the same for all clients on the same device?